app.rb

```diff
@@ -0,0 +1,57 @@
+require 'pry'
+require 'active_record'
+
+## Connect to the database
+require_relative 'db/connection'
+
+# Load models
+require_relative 'models/apartment'
+require_relative 'models/tenant'
+
+require 'sinatra'
+require 'sinatra/reloader'
+
+get '/' do
+  erb :"index"
+end
+
+get '/apartments' do
+  @apartments = Apartment.all
+  erb :"apartments/index"
+end
+
+get '/apartments/:id' do
+  @apartment = Apartment.find(params[:id])
+  erb :"apartments/show"
+end
+
+post '/apartments/new' do
+  Apartment.create(address:params[:apartmentAddress], monthly_rent:params[:apartmentMonthlyRent], sqft:params[:apartmentSqft], num_beds:params[:apartmentNumBeds], num_baths:params[:apartmentNumBaths])
+  redirect "/apartments"
```

Comment:
A slightly cleaner way to do this is to use nested params. See the corresponding comment on `apartments/index` This would allow you to instead just say
```rb
Apartment.create(params[:apartment])
```